SHARDING.md

```diff
@@ -322,16 +321,18 @@ the latest TrustedRoot. In practice for our TUF root with timestamp validity
 of one week, this means setting the validity window to be at least one week
 past when the TUF targets are signed.
 
-You will need three timestamps:
+You will need four timestamps:
 
 * New shard's start time for TrustedRoot, which should be when the shard was spun up.
-* New shard's start time for SigningConfig, which should be one week past when the root-signing PR is merged
-and new TUF repository is published. Overestimate (by an hour or even a day) if needed,
-as you don't want clients to start using the new shard before all clients
-have picked up the latest TrustedRoot.
 * Previous shard's end for TrustedRoot: Clients will consider this shard's entry timestamps invalid
 if they are after this date. This must be after the start of the new shard. It doesn't have to be
 overly precise, and can be updated in a future TUF signing event.
+* New shard's start time for SigningConfig, which should be one week past when the root signing PR is merged
+and new TUF repository is published. Overestimate by at least a day of the expected merge date,
+as you don't want clients to start using the new shard before all clients
+have picked up the latest TrustedRoot. For production, the overestimate should be longer than staging,
```

Comment:
I think longer than staging is not a great recommendation. Should we pick something concrete and hold ourselves to it?